src/models/launch_heavy_1/README.md

```diff
@@ -0,0 +1,47 @@
+# Launch Heavy (launch\_heavy\_1)
+
+- [External Overview](./external-overview.md)
+- [Customization & Repairs](./repairs.md)
+
+![Launch Heavy](./img/launch-heavy.webp)
+
+The System76 Launch Heavy is a configurable keyboard with the following specifications:
+
+- Chassis
+  - [Open-source milled chassis design](https://github.com/system76/launch/tree/master/chassis/launch-heavy)
+  - Dual magnetically-attachable 15-degree angle lift bars
+- Electronics
+  - [Open-source PCB design](https://github.com/system76/launch/tree/master/pcb-heavy)
+  - Individually addressable RGB LED backlighting
+  - N-key rollover
+- Sockets and Switches
+  - Kailh MX hot swap sockets
+  - Default switch options:
+    - NovelKeys x Kailh Box Jade
+    - NovelKeys x Kailh Box Royal
+    - Kailh Box Silent Brown
+    - Kailh Box Silent Pink
+- Keycaps
+  - PBT plastic
+  - Dye sublimated legend
+  - XDA profile
+  - Sizes:
+    - 6x 2U keycaps (both `Space` bars, left `Shift`, numpad `0`, numpad `+`, and numpad `Enter`)
+      - All six 2U caps use plate-mount stabilizers
+    - 8x 1.5U keycaps (`Tab`, `Caps`, left `Ctrl`, right `Fn`, right `Shift`, `Enter`, `Bksp`, and `Del`)
+    - 91x 1U keycaps
+- Layout
+  - Modified ANSI US QWERTY
+- Custom Configurations
+  - Customize layout and lighting in firmware with the [System76 Keyboard Configurator](https://github.com/pop-os/keyboard-configurator)
+  - [Available on Linux, Windows, and macOS](https://system76.com/accessories/launch/download)
+- Integrated Hub
+  - 2x USB 3.2 Gen 1 Type-C
+  - 2x USB 3.2 Gen 1 Type-A
+- Firmware
+  - [Open-source QMK-based firmware](https://github.com/system76/qmk_firmware/tree/master/keyboards/system76/launch_heavy_1/)
+- Included cables:
+  - 1x USB Type-C to USB Type-C
+  - 1x USB Type-C to USB Type-A
+- Dimensions
+  - 39.40cm x 13.51cm x 3.31cm, 1315.42g
```

Comment:
> 39.40cm x 13.51cm x 3.31cm, 1315.42g

The website's product card and tech specs both currently list the height as 3.03 cm (30.3 mm), so that still either needs to be fixed on the sales page or fixed here.

Reply:
Fixed in https://github.com/system76/tech-docs/pull/157/commits/88318735f7c876dc5da2cc93cee0af6f2a8bbece